Hyeonwoo-Park/BOJ2629/Scale.java

```diff
@@ -0,0 +1,41 @@
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Set;
+import java.util.StringTokenizer;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+public class Scale {
+
+	public static void main(String[] args) throws Exception {
+		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+		int N = Integer.parseInt(reader.readLine());
+		StringTokenizer token = new StringTokenizer(reader.readLine());
+		Set<Integer> s = new CopyOnWriteArraySet<>();
+		s.add(0);
+
+		for (int i = 0; i < N; ++i) {
+			int inp = Integer.parseInt(token.nextToken());
+
+			for (Integer num : s) {
+				if (num - inp > 0)
+					s.add(num - inp);
+				else
+					s.add(inp - num);
```

Comment:
```suggestion
				s.add(Math.abs(num - inp));
```
abs() 쓰면 간단히 표현 가능해요~!